bayesian/dependency_finder.py

```diff
@@ -104,4 +114,7 @@ def _flatten(deps, collect):
                 out["details"][0]['_resolved'] = resolved_deps
             result.append(out)
 
+        if source == 'osio':
```

Comment:
storing of contents should be per manifest. This loop should be part of main for loop and not outside.